pkg/types/clustermetadata.go

```diff
@@ -9,7 +9,8 @@ import (
 // ClusterMetadata contains information
 // regarding the cluster that was created by installer.
 type ClusterMetadata struct {
-	ClusterName             string `json:"clusterName"`
+	ClusterName string `json:"clusterName"`
```

Comment:
`gofmt` [doesn't like this][1].

[1]: https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/783/pull-ci-openshift-installer-master-gofmt/1529/build-log.txt